utility/src/main/java/org/oppia/android/util/logging/performancemetrics/ProdPerformanceMetricsUtils.kt

```diff
@@ -0,0 +1,88 @@
+package org.oppia.android.util.logging.performancemetrics
+
+import android.app.ActivityManager
+import android.content.Context
+import android.net.TrafficStats
+import org.oppia.android.app.model.OppiaMetricLog
+import org.oppia.android.util.logging.ConsoleLogger
+import java.io.File
+import javax.inject.Inject
+
+/** Utility to extract performance metrics from the underlying Android system. */
+class ProdPerformanceMetricsUtils @Inject constructor(
```

Comment:
Suggest instead: ``PerformanceMetricsUtilsImpl`` for parity with other such classes.

Reply:
changed to `PerformanceMetricsAssessorImpl` in parity with interface's renaming.